extensions/Worker.Extensions.Http.AspNetCore/src/FunctionsHostBuilderExtensions.cs

```diff
@@ -72,6 +72,8 @@ internal static IHostBuilder ConfigureAspNetCoreIntegration(this IHostBuilder bu
                     // By default, the X-Forwarded-For, X-Forwarded-Host, and X-Forwarded-Proto headers
                     // are sent by the host and will be processed by the worker.
                     options.ForwardedHeaders = ForwardedHeaders.All;
+
+                    options.ForwardLimit = null;
```

Comment:
`null` removes any limits -- this is a placeholder for now. Do we want an explicitly chosen value for this limit?